### ISSUE TITLE
Cannot purchase

### ISSUE THREAD

**Issue by Yuval0815** [oodesign/merge-duplicate-symbols#190]
Hello,

I wish to purchase the plugin for my team but unfortunately it says my credit card cannot be charged. I tried with 3 different ones. 

Thanks,
-Yuval

**Comment by oodesign** [comment]
Hi @Yuval0815 ,

We use Gumroad as our purchase platform, and therefore I'm afraid we need to stick to its [payment options](https://customers.gumroad.com/before-you-buy/a-guide-to-buying-on-gumroad).

Could you please take a look [their documentation](https://customers.gumroad.com/before-you-buy/a-guide-to-buying-on-gumroad) and check if any of the options would work for you? Or otherwise, please send them a message (there's an option to do so on the bottom-right corner) describing your case.

Hope it helps a bit!

Thanks for reaching out!

**Comment by Yuval0815** [comment]
Hey and thanks for the reply. I contacted their support, hope we can resolve this issue. 

Thanks

**Comment by oodesign** [comment]
Great! Please let us know how that goes, and if there is anything we can help with 😊

**Comment by Yuval0815** [comment]
Hello again,

We tried and It seems Gumroad support cannot / will not help us (tirelessly claim the problem is on our end). We tried several bank accounts from our team and it seems very unlikely that all of them declined the purchase. Maybe it's country-restricted? Is there any other way we can purchase the plugin from you directly or through another platform? It will help us greatly.

Thanks,
-Yuval
